collections/_version2-english/9-fork/2-to-dos.md

```diff
@@ -0,0 +1,70 @@
+---
+title: To-do's
+---
+{% capture glossary_path %}{{ "/" | absolute_url }}/{{page.collection}}/7-glossary.html{% endcapture %}
+{% assign glossary_child_universe = glossary_path | append: '#Child_Universe' %}
+{% assign glossary_final_outcome = glossary_path | append: '#Final_Outcome' %}
+{% assign glossary_finalized_market = glossary_path | append: '#Finalized_Market' %}
+
+{% assign glossary_fork = glossary_path | append: '#Fork' %}
+{% assign glossary_forking_market = glossary_path | append: '#Forking_Market' %}
+{% assign glossary_forking_period = glossary_path | append: '#Forking_Period' %}
+{% assign glossary_locked_universe = glossary_path | append: '#Locked_Universe' %}
+{% assign glossary_losing_universe = glossary_path | append: '#Losing_Universe' %}
+{% assign glossary_parent_universe = glossary_path | append: '#Parent_Universe' %}
+{% assign glossary_reputation_token = glossary_path | append: '#Reputation_Token' %}
+{% assign glossary_repv1 = glossary_path | append: '#REPv1' %}
+{% assign glossary_repv2 = glossary_path | append: '#REPv2' %}
+{% assign glossary_roi = glossary_path | append: '#ROI' %}
+{% assign glossary_sibling_universe = glossary_path | append: '#Sibling_Universe' %}
+{% assign glossary_winning_universe = glossary_path | append: '#Winning_Universe' %}
+
+# To-do's
+If you are a REP holder, there are some things you **MUST** do for a [fork]({{glossary_fork}}) to not lose your [REP]({{glossary_reputation_token}}).
+
+## Before a fork starts
+REP holders have 2 things to do even before a [fork]({{glossary_fork}}) starts so that they don't lose the value of their [REP]({{glossary_reputation_token}}). One is to [convert REPv1 to REPv2](#convert-repv1-to-repv2) and the other is to [check monthly to see if a fork occurs](#check-monthly-to-see-if-a-fork-occurs).
```

Comment:
You don't have to convert REPv1 to REPv2 before a fork *starts*, you just have to do it before the fork *ends*.

Reply:
I submitted a issue https://github.com/msagansk/Augur.Guide/issues/38 .

Reply:
The issue https://github.com/msagansk/Augur.Guide/issues/38 was closed, so I'm closing this conversation.